spi/src/main/java/org/opensearch/jobscheduler/spi/LockModel.java

```diff
@@ -173,7 +174,7 @@ public XContentBuilder toXContent(final XContentBuilder builder, final Params pa
 
     @Override
     public String toString() {
-        return Strings.toString(this, false, true);
+        return Strings.toString(XContentType.JSON, this, false, true);
```

Comment:
It's not obvious to me: was the old behavior of `Strings.toString()` using JSON?

Reply:
No, the parameters have been changed recently due to this PR : https://github.com/opensearch-project/OpenSearch/pull/6009#event-8357983015